.npmignore

```diff
@@ -1,10 +1,17 @@
+/.vscode
 .github/
 .vscode/
 azure-pipelines/
 node_modules/
 build/
 lib/
+src/
```

Comment:
I'm not sure why all the files are being ignored here.
If `src/` isn't included in the package, then node-gyp cannot build the package.